content/en/tutorials/migrate_gcp_122.md

```diff
@@ -0,0 +1,18 @@
+# Upgrading Google Kubernetes Engine Version
```

Comment:
can the file name not include the version, so we don't need to add a new page for each version in the future?
we could have many versions in the same page and use this https://learn.netlify.app/en/shortcodes/expand/